apparmor.d/profiles-g-l/lsd

```diff
@@ -0,0 +1,22 @@
+# apparmor.d - Full set of apparmor profiles
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{,usr/}{,local/}bin/lsd
```

Comment:
What is lsd? If it is this [new ls](https://github.com/Peltoche/lsd), you should not confine it at all.

Reply:
It's a third-party program, why it should be trusted?

Reply:
Yeah, that first attempt was bad, but how custom `ls` could break the system? It's not replacing `ls` - it won't break scripts. Pipes? User could modify `local` drop-in for edge cases. If we'll account for pipes and edge cases half of profiles would be broken...